packages/api/pages/api/users/[userId].ts

```diff
@@ -0,0 +1,59 @@
+import * as z from 'zod';
+import { SystemRole, UpdateUserRequestBody } from '@translation/api-types';
```

Comment:
This occurs when I try to update the role of my own user to "Admin".  
```
Module not found: Can't resolve '@translation/api-types'
  1 | import * as z from 'zod';
> 2 | import { SystemRole, UpdateUserRequestBody } from '@translation/api-types';
  3 | import { authorize } from '../../../shared/access-control/authorize';
  4 | import createRoute from '../../../shared/Route';
  5 | import { client } from '../../../shared/db';

https://nextjs.org/docs/messages/module-not-found
```
I'm not sure what's going on, because I can clearly see `@translation/api-types` in `tsconfig.base.json`.

Reply:
Can you try running `nx reset` and then run the servers again?

Reply:
It still occurs. I tried restarting my computer and doing it again, still happened. I tried deleting the `npm_modules/.cache` directory, did it again, and the error still occurs. I'll keep trying next week.

Reply:
It's strange that it only occurs there, because many other files import `@translation/api-types`.

Reply:
I tried temporarily to change that line to  `import { SystemRole, UpdateUserRequestBody } from '../../../../api-types/src/index';`. Then I got a 403 error because I don't have permission to update the role of a user. I have to manually set myself as an admin in the DB? How do I do that? I tried and got errors.

Reply:
Use this script:
```sql
INSERT INTO "public"."UserSystemRole" ("userId", "role")
VALUES ('<user-id>', 'ADMIN');
```

Reply:
So, I was finally able to reproduce it. I traced the issue to the import of SystemRole. For some reason because that is not just a typescript interface it creates an issue

Reply:
Ok, I upgraded nx to the latest version, and the issue seemed to go away. Can you try again and confirm?

Reply:
It worked without a hitch this time. Thanks!